File: Java/0080_remove_duplicates_from_sorted_array_ii/0080_remove_duplicates_from_sorted_array_ii.java
class Solution {
    public int removeDuplicates(int[] nums) {
        if (nums.length < 2) return nums.length;
        int i = 0;
        for (int j = 0; j < nums.length; j++) {
            if (i < 2 || nums[j] != nums[i - 2]) {
                nums[i] = nums[j];
                i++;
            }
        }
        return i;
    }
}

File: Python/0080_remove_duplicates_from_sorted_array_ii/0080_remove_duplicates_from_sorted_array_ii.py
class Solution:
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        if len(nums) < 2:
            return len(nums)
        i = 0
        for j in range(len(nums)):
            if i < 2 or nums[j] > nums[i - 2]:
                nums[i] = nums[j]
                i += 1
        return i
